ONSdigital/eq-author-app
# Remove storybook

### What is the context of this PR?
- Removes references to storybook

### How to review 
- Ensure tests pass
- Ensure all references have been removed
- Ensure Netlify does not deploy storybook
